rusk-recovery/src/state.rs

```diff
@@ -17,11 +17,16 @@ use tracing::info;
 use url::Url;
 
 use execution_core::{
-    stake::{StakeAmount, StakeData},
-    BlsPublicKey, JubJubScalar, Note, PublicKey, Sender,
+    license::LICENSE_CONTRACT,
+    signatures::bls::PublicKey as AccountPublicKey,
```

Comment:
Example here

Reply:
I did that deliberately since we are now aliasing the `bls::PublicKey` in each file new, I took the freedom to name it `AccountPublicKey` in files it is only used for moonlight-accounts. Do you find it confusing?

Reply:
I don't find it confusing at all, more that I thought we'd settled on not using the alias. That said, its only in the place where they're used so I'm ok with it.